src/components/FAQ/FAQ.scss

```diff
@@ -1,38 +1,56 @@
+.page-title {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
 .faq-section {
   display: flex;
   flex-direction: column;
-  padding-top: 110px;
-  .row {
-    display: flex;
-    justify-content: space-evenly;
-    flex-wrap: wrap;
-    .module {
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      margin: 0px 30px 60px;
-    }
-    .title {
-      background: #EBEBEB;
-      border-radius: 25px;
-      font-size: 35px;
-      line-height: 34px;
-      text-align: center;
-      color: #000000;
-      width: 478px;
-      padding: 20px 30px;
-      cursor: pointer;
-    }
-    .answer {
-      background: rgba(73, 73, 73, 0.2);
-      font-size: 25px;
-      line-height: 24px;
-      text-align: center;
-      color: #413F3F;
-      padding: 40px;
-      width: 435px;
-      transition: visibility 0.5s, opacity 0.5s linear;
-    }
+}
+
+.faq-row {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+.module {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: auto;
+  padding-bottom: 30px;
+  width: 69%;
+}
+
+.faq-title {
+  display: flex;
+  background: #EBEBEB;
+  border-radius: 25px;
+  font-size: 35px;
+  line-height: 34px;
+  color: #000000;
+  width: 95vw;
+  max-width: 1053px;
+  padding: 5%;
+  cursor: pointer;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.faq-answer {
+  font-size: 22px;
+  line-height: 30px;
+  text-align: center;
+  font-family: "Nunito", sans-serif;
+  color: #FFFFFF;
+  width: 93vw;
+  max-width: 1036.5px;
+  display: none;
+  padding-top: 21px;
+}
```

Comment:
What's great about SCSS is that we can nest these styles now. Take a look at how they were implemented before and see if you can nest them now.